lib/packagers/yarn.test.js

```diff
@@ -80,10 +78,6 @@ describe('yarn', () => {
       const expectedResult = {
         problems: [],
         dependencies: {
-          bestzip: {
-            version: '2.1.5',
-            dependencies: {}
-          },
```

Comment:
Can't you leave this file untouched? I think it has nothing to deal with the purpose of the PR.
That test is only about dependencies not related to building the zip file.

Same for all the `tests/mocks/*.mock.json` files